src/FrigadeCarousel/styled.ts

```diff
@@ -107,26 +114,26 @@ export const CompletedPill = styled.div`
 `
 
 export const H3 = styled.p`
-  font: bold 18px/22px Inter;
+  font: bold 18px/22px sans-serif;
   letter-spacing: calc(18px * -0.01);
   margin: 0;
 `
 
 export const H4 = styled.p`
-  font: bold 16px/20px Inter;
+  font: bold 16px/20px sans-serif;
   letter-spacing: calc(16px * -0.01);
   margin: 0;
 `
 
 export const Body = styled.p`
-  color: #000;
-  font: normal 14px/22px Inter;
+  color: ${({ theme }) => theme.colorText};
+  font: normal 14px/22px sans-serif;
   margin: 0;
 `
 
 export const Small = styled.p`
-  color: #000;
-  font: 600 12px/16px Inter;
+  color: ${({ theme }) => theme.colorText};
+  font: 600 12px/16px sans-serif;
```

Comment:
Any reason for the specific font here? Ditto above on line 130

Reply:
TIL about how finicky the `font` shorthand is in CSS. You can't use `font` without specifying a value for `font-family`, and despite `inherit` being a legal value, you can't use `inherit` in the shorthand, so I fell back to the basic `sans-serif` keyword, which _should_ be overridden by anything set higher up in the cascade.

I could also just switch this to be `font-size: x; font-weight: y; line-height: z` if you think there's a chance it might break.

Reply:
Cool, let's give it a shot! Good to know